OWNERS

```diff
@@ -1,4 +1,4 @@
-approvers:
```

Comment:
Cannot parse file: error converting YAML to JSON: yaml: line 6: could not find expected ':'.